plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -58,6 +58,7 @@
 import java.util.Map;
```

Comment:
Please provide a tests where the new session property is used and where it works as expected. For example you can set it 0 to see it fails and set it to 1 and use `TestingPostgreSqlServer#getRemoteDatabaseEvents` to see that only one insert was done.